gen3-client/g3cmd/retry-upload.go

```diff
@@ -53,6 +53,8 @@ func retryUpload(failedLogMap map[string]commonUtils.RetryObject) {
 	var guid string
 	var presignedURL string
 	var err error
+	var bucketPtr string
```

Comment:
This is suffixed with Ptr but it doesn't look like it's a pointer? It's just a string